### DISCUSSION TITLE
6.11.4 - Quotes missing in data result

### DISCUSSION THREAD

**Discussion by danielbattat** [react-hook-form/react-hook-form#3830] [Q&A]
Starting in 6.11.4, quotes in the field name get stripped out of the result.

This is problematic as the server expects a certain field name.

Here is a csb - changing to 6.11.3 shows that the quote appears as expected:
https://codesandbox.io/s/react-hook-form-controller-as-prop-forked-6ton9

![image](https://user-images.githubusercontent.com/30219620/103577283-2d51b300-4edd-11eb-933b-0fc0b47b7277.png)

What this by design, or is something broken? Any insight appreciated.

Thank you!

**Comment by bluebill1049** [comment]
can you update the codesandbox?

**Reply by danielbattat** [reply]
Sorry about that!
https://codesandbox.io/s/react-hook-form-controller-as-prop-forked-sj03p
